lib/components/src/ScrollBar/components/TrackHorizontal.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC, useCallback, useRef } from 'react';
+import { document as _document } from 'global';
+import * as Styled from '../styled';
+import { ScrollAreaTrackProps } from '../types';
+
+const document = _document as Document;
```

Comment:
why? just for the types?

Reply:
Yes - the global we are using is however pretty bad, and I am proposing in another PR that we use [window-or-global](https://www.npmjs.com/package/window-or-global) which has nice types, 0 dependencies and is maintained